CODE_OF_CONDUCT.md

```diff
@@ -1,9 +1,3 @@
-<!-- Copyright Contributors to the Qiskit project. -->
-
 # Code of Conduct
-All members of this project agree to adhere to the Qiskit Code of Conduct listed at [https://github.com/Qiskit/qiskit/blob/master/CODE_OF_CONDUCT.md](https://github.com/Qiskit/qiskit/blob/master/CODE_OF_CONDUCT.md)
-
-----
 
-License: [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/),
-Copyright Contributors to Qiskit.
```

Comment:
@1ucian0 do you know why we had this? We normally use the Apache license. Were we copyrighting the license text itself? Qiskit SDK repo doesn't have it.